test/src/test/java/org/corfudb/integration/CorfuReplicationMultiSinkIT.java

```diff
@@ -73,119 +73,4 @@ public void testRoleChange() throws Exception {
         log.info("Testing after role change");
         verifySnapshotAndLogEntrySink(true);
     }
-
-    /*/**
-     * Verify snapshot sync in a topology with 3 Sink clusters and 1 Source cluster.
-     * Then perform a role switch by changing the topology to 3 Source clusters, 1 Sink cluster and verify that the new
-     * Sink cluster successfully completes a snapshot sync.
-     * @throws Exception
-     *//*
-    @Test
-    public void testRoleChange() throws Exception {
```

Comment:
Why are these tests deleted?

Reply:
They were already in a commented-out form.  The commented test is already there - testRoleChange() but most of its implementation and helper methods were moved to MultiSourceSinkIT in an earlier PR as they were common with CorfuReplicationMultiSourceIT.